propertyestimator/tests/test_workflow_protocols.py

```diff
@@ -0,0 +1,13 @@
+"""
+Units tests for propertyestimator.workflow
+"""
+
+def test_protocol_decorators():
```

Comment:
Is this a test will be implemented later, or should we scrap this file?

Reply:
This will be a test implemented in my testing push next week (along with many others in this file), so it's probably safe to keep the file for now.